### PR TITLE
Updating parsed plugin list from plugins.yml

### PR DESCRIPTION
**Pull request by nanliu**

### CODE REVIEW

File: assets/catalog/parsed_plugin_list.js [hunk 1]
```diff
@@ -331,6 +367,12 @@ myfcn(
     "url": "https://github.com/intelsdi-x/snap-plugin-publisher-hana"
   },
   {
+    "name": "Heapster",
+    "type": "Publisher",
```
Comment (mjbrender) [inline assets/catalog/parsed_plugin_list.js]: minor typo

File: assets/catalog/parsed_plugin_list.js [hunk 2]
```diff
@@ -331,6 +367,12 @@ myfcn(
...
+    "description": "Publisher exposing container metrics through a kubelet-compatible API",
+    "url": "https://github.com/intelsdi-x/snap-plugin-publisher-heapster"
+  },
+  {
     "name": "HEKA",
```
Comment (mjbrender) [inline assets/catalog/parsed_plugin_list.js]: I learned that HEKA Is actually Heka

File: assets/catalog/parsed_plugin_list.js [hunk 3]
```diff
@@ -187,6 +193,12 @@ myfcn(
     "url": "https://github.com/intelsdi-x/snap-plugin-collector-mesos"
   },
   {
+    "name": "Mongodb",
```
Comment (mjbrender) [inline assets/catalog/parsed_plugin_list.js]: could we capitalize to MongoDB ?

File: assets/catalog/parsed_plugin_list.js [hunk 4]
```diff
@@ -265,6 +277,12 @@ myfcn(
     "url": "https://github.com/intelsdi-x/snap-plugin-collector-scaleio"
   },
   {
+    "name": "Schedstat",
+    "type": "Collector",
+    "description": "Collects Linux scheduler metrics from /proc/schedstats",
+    "url": "https://github.com/intelsdi-x/snap-plugin-collector-schedstat"
+  },
+  {
     "name": "Snmp",
```
Comment (mjbrender) [inline assets/catalog/parsed_plugin_list.js]: Could we make SNMP?

File: assets/catalog/parsed_plugin_list.js (hunk 1 above)
Comment (nanliu) [inline assets/catalog/parsed_plugin_list.js]: What's the typo?

**Pull request closed without merging**